integration-tests/src/test/resources/features/error-handling.feature

```diff
@@ -125,3 +125,54 @@ Feature: Error handling tests
     }
     """
     Then Webhook site with id "${env.webhook.site.uuid}" contains request with text ""id":"${cloud-event.error-handling-update-test.id}","source":"mySource","type":"myType","subject":"mySubject"" within 1 minute
+
+  @errorhandlingdlqheaders
+  Scenario: A failure occurs and a cloud event message is sent by managed connector to DLQ with correct headers
+    Given authenticate against Manager
+    And create a new Bridge with body:
+    """
+    {
+        "name": "ehBridge",
+        "cloud_provider": "aws",
+        "region": "us-east-1",
+        "error_handler": {
+            "type": "webhook_sink_0.1",
+            "parameters": {
+                "endpoint": "https://webhook.site/${env.webhook.site.uuid}"
+            }
+        }
+    }
+    """
+    And the Bridge "ehBridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "ehBridge" is available within 2 minutes
+
+    And add a Processor to the Bridge "ehBridge" with body:
+    """
+    {
+       "name": "sqsProcessor",
+       "source": {
+          "type": "aws_sqs_source_0.1",
+          "parameters": {
+                "aws_queue_name_or_arn": "${env.aws.queue-name-or-arn}",
+                "aws_region": "${env.aws.region}",
+                "aws_access_key" : "${env.aws.access-key}",
+                "aws_secret_key" : "${env.aws.secret-key}"
+          }
+       }
+    }
+    """
+    And the Processor "sqsProcessor" of the Bridge "ehBridge" is existing with status "ready" within 3 minutes
+    And the Processor "sqsProcessor" of the Bridge "ehBridge" has source of type "aws_sqs_source_0.1"
+
+    When send a message with text "invalid message for bridge ${bridge.ehBridge.id}" to sqs queue
+
+    Then Webhook site with id "${env.webhook.site.uuid}" contains request with text "invalid message for bridge ${bridge.ehBridge.id}" within 1 minute and headers:
+      | x-rhose-processor-id          | ${bridge.ehBridge.processor.sqsProcessor.id} |
+      | x-rhose-original-event-source | RHOSE                                        |
+      | x-rhose-original-event-id     |                                              |
+      | x-rhose-error-id              | OPENBRIDGE-6                                 |
+      | x-rhose-bridge-id             | ${bridge.ehBridge.id}                        |
+      | x-dead-letter-topic           |                                              |
+      | x-dead-letter-reason          |                                              |
+      | x-dead-letter-partition	      |                                              |
```

Comment:
```suggestion
      | x-dead-letter-partition	    |                                              |
```

Reply:
sorry - can't see that extra space here. On GH it looks fine, at least for me - let me know

Reply:
@afalhambra I guess it is a difference in browsers (I use Firefox). There is a tab character right after `x-dead-letter-partition`, should be replaced by spaces.
```suggestion
      | x-dead-letter-partition        |                                              |
```

Reply:
let me know if it's fine now

Reply:
Looks fine